charts/brigade/templates/gateway-cr-deployment.yaml

```diff
@@ -17,6 +17,7 @@ spec:
     metadata:
       labels:
         app.kubernetes.io/name: {{ template "brigade.fullname" . }}
+        app: {{ template "brigade.fullname" . }}
```

Comment:
A few other instances of needed backwards-compatible label additions; ref https://github.com/Azure/brigade/pull/707